modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -133,6 +133,8 @@ Prior to Redpanda version 24.2, this meant that some cores on a broker could ina
 
 Starting in v24.2, topic-aware intra-broker partition balancing allows for dynamically reassigning partitions within a broker.  Redpanda prioritizes an even distribution of a topic's partition replicas across all cores in a broker. If a broker's core count changes, when the broker starts back up, Redpanda can check partition assignments across the broker's cores and reassign partitions, so that a balanced assignment is maintained across all cores. Redpanda can also check partition assignments when partitions are added to or removed from a broker, and rebalance the remaining partitions between cores.
 
+NOTE: Changing the number of CPU cores in a running cluster is supported from v24.2 only, and requires that you enable `node_local_core_assignment` as described in the previous note.
```

Comment:
@daisukebe @wzzzrd86 for this note are we referring to a _manual_ change in number of cores? I am not sure if there is some process that can dynamically change the number and if so, if that's worth mentioning.

Reply:
@kbatuigas Yes, this is meant to be a manual change in number of cores. To be clear, the general step is below:

* Stop a broker, which originally uses 10 cores.
* Start the broker with specifying 9 cores (whatever).

This has not been working as expected since beginning but this just works today. Does this answer your question?